tools/harness-thci/OpenThread.py

```diff
@@ -71,20 +73,28 @@ def __init__(self, **kwargs):
             self.handle = None
             self.AutoDUTEnable = False
             self._is_net = False                  # whether device is through ser2net
-            self.logStatus = {'stop':'stop', 'running':'running', "pauseReq":'pauseReq', 'paused':'paused'}
-            self.joinStatus = {'notstart':'notstart', 'ongoing':'ongoing', 'succeed':'succeed', "failed":'failed'}
+            self.logStatus = {
+                'stop': 'stop',
+                'running': 'running',
+                spauseReqs: 'pauseReq',
```

Comment:
what is this `spauseReqs`? I cannot find its definition. Is it meant to `'pauseReq'`?

Reply:
carelessly wrong op during replacement. updated.